### DISCUSSION TITLE
Dynamic simulation automation

### DISCUSSION THREAD

**Discussion by BFAnas** [DanWBR/dwsim#556] [Q&A] [answered]
Hi @DanWBR,

Can you please provide a code snippet for performing dynamic simulation automation in python, it would be particularly interesting to show how we can start/stop/reset and perform a step in the simulation, how to setup the simulation step and duration, and how to have a controller in the loop that can change the value of a manipulated variable.

Thanks!

**Comment by DanWBR** [comment, marked as answer]
@BFAnas I have developed a separate application/library to do this, but so far it has not been released to the public, sorry about that.

**Reply by BFAnas** [reply]
Hi @DanWBR , thanks for the answer! Are you planning to release it to the public? if yes when?
